app/api/items/testdata/default.yaml

```diff
@@ -0,0 +1,17 @@
+server:
```

Comment:
Wow, we are not gonna replicate the default config file in every directory that we test, this is not gonna work, right?